pyproject.toml

```diff
@@ -24,7 +24,7 @@ dependencies = [
   "asyncssh~=2.19.0",
   'circus~=0.19.0',
   'click-spinner~=0.1.8',
-  'click~=8.1',
+  'click~=8.1.0',
```

Comment:
Can you recall why we need `8.1.0` and not higher `8.1`? It seems to me generally that we should not pin patches, as these should ideally not break anything.

Reply:
This doesn't pin the patch, note the tilde. (I am also always confused about this : 😅

Reply:
right, this is not the first time I am confusing it^^